test/functional-tests/pageObjects/ccd-components/fields/ccdYesNoField.js

```diff
@@ -15,12 +16,13 @@ class CcdYesNoField {
    */
   constructor(css){
     this.css = css;
-    this.yesCss = `${css} .form-group > div:nth-of-type(1) input`;
-    this.noCss = `${css} .form-group > div:nth-of-type(2) input`;
+    this.yesCss = `${css} .form-group #YesNoField-Yes`;
```

Comment:
nice abstraction, should we delete this.noCss/this.yesCss and parse the selector straight into the new RadioField if we are not using those variables anywhere else

Reply:
Yes will do.